<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortSettingsForm</class>
 <widget class="QWidget" name="PortSettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>端口号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmb_Port"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>波特率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmb_Baud">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btn_SerialSet">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>更多设置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_OpenSerial">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>打开串口</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btn_ClearSend">
       <property name="text">
        <string>清空发送区</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ClearRecv">
       <property name="text">
        <string>清空接受区</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_SendNewLine">
     <property name="text">
      <string>发送新行</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_SendHex">
     <property name="text">
      <string>16进制发送</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_RecvToFile">
     <property name="text">
      <string>接收到文件</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_AddTime">
     <property name="text">
      <string>增加时间戳</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_HexShow">
     <property name="text">
      <string>Hex显示</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_ShowSend">
     <property name="text">
      <string>发送回显</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btn_saveData">
     <property name="text">
      <string>保存接收显示框数据</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
